.github/workflows/scripts/pending-prs

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+export BROWSER=echo
```

Comment:
This is a nasty hack to capture the url for filtering out the PRs that we're actually querying so that they can be opened in a browser. `gh -w` actually opens the link directly, so rather than doing that I just want to echo it out and capture its value 😅